testing/scripts/capture_zoneid_snapshots.py

```
@@ -0,0 +1,132 @@
+            await capture_state(page, state="open",    name="ipv6-zoneid-open")
+            await capture_state(page, state="success", name="ipv6-zoneid-success")
+            await capture_state(page, state="warning", name="ipv6-zoneid-warning")
+            await capture_state(page, state="error",   name="ipv6-zoneid-error")
+
+            await browser.close()
+        return 0
+    finally:
+        server.terminate()
+        try:
+            server.wait(timeout=2)
+        except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)